Project 3 (15.12)/script.js

```diff
@@ -0,0 +1,40 @@
+const button = document.querySelector('.generate-table');
+
+button.addEventListener('click', createTable);
+
+function createTable() {
+	console.log('test');
+	const rows = document.querySelector('.rows-number').value;
+
+	const columns = document.querySelector('.columns-number').value;
+
+	const table = document.querySelector('table');
+
+	table.innerText = '';
+	if (rows < 0) {
+		alert('Only plus numbers');
+	}
+
+	for (let i = 0; i < rows; i++) {
+		const r = table.insertRow(i);
+		console.log(r);
+
+		for (let c = 0; c < columns; c++) {
+			const y = r.insertCell(c);
+			y.innerText = 'Row-' + i + ' ' + 'Column-' + c;
```

Comment:
dałbym +1 do indexu, bo teraz masz np "row 0 column 0" :) nie jest to błąd, ale wiesz, inaczej brzmi pierwsza kolumna, a inaczej "zerowa" :)